Yoast/wordpress-seo
# Add is false check

## Context


* Fixes a case we false was given and accessed as an object.

## Summary


This PR can be summarized in the following changelog entry:

* Fixes a bug where a boolean variable was accessed as an object resulting in a non-object property retrieval notice.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Unfortunately I have no proper steps to reproduce this issue.
* I triggered the error by returning false immediately in https://github.com/Yoast/wordpress-seo/blob/trunk/src/builders/indexable-post-builder.php#L76
* Checkout the release branch
* Alter the file builders/indexable-post-builder.php
* Remove all indexables and visit a post on the frontend. An error will be given
* Checkout this branch
* Change the file builders/indexable-post-builder.php again
* There error should be gone. 

## UI changes

## Documentation

## Quality assurance


Fixes #14965
